src/server/seeds/dev/012_00_waypoints.js

```diff
@@ -0,0 +1,31 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("waypoints")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("waypoints").insert([
+        {
+          id: 1,
+          route_id: 1,
+          port_id: 1,
+          lon: 2344.44,
+          lat: 2234.87
+        },
+        {
+          id: 2,
+          route_id: 2,
+          port_id: 2,
+          lon: 2344.44,
+          lat: 2234.87
+        },
+        {
+          id: 3,
+          route_id: 3,
+          port_id: 3,
+          lon: 2344.44,
+          lat: 2234.87
+        }
```

Comment:
We need waypoints associated with all of the routes. And we also need waypoints for the ports, but use the json file I linked above for that.

Also note, that route waypoints will have port_id set to null and port waypoints will have route_id set to null.